WordPress/Classes/Models/TimezoneInfo.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  TimezoneInfo+CoreDataClass.swift
+//  WordPress
+//
+//  Created by Asif on 17/11/17.
+//  Copyright © 2017 WordPress. All rights reserved.
+//
+//
+
+import Foundation
```

Comment:
Would you mine removing the first 9 lines of this file? We don't include file headers on this project. Thanks